src/main/java/uk/gov/ons/ctp/response/action/service/ActionPlanService.java

```diff
@@ -64,18 +64,16 @@ public ActionPlan findActionPlanByName(final String name) {
       readOnly = false,
       timeout = TRANSACTION_TIMEOUT)
   public ActionPlan createActionPlan(final ActionPlan actionPlan) {
-    log.debug(
-        "Creating action plan, Name: {}, Selectors: {}",
-        actionPlan.getName(),
-        actionPlan.getSelectors());
+    log.with(actionPlan.getSelectors())
```

Comment:
need a key name

Reply:
`.with()` handles the object so will show up in logs as `collection_exercise_id = uuid active_enrolment = true` is it worth putting "selectors" as a key?

Reply:
Never mind going to put the key in, the keys in the selectors object are not guaranteed to be  collection_exercise_id and active_enrolment.